python-threatexchange/threatexchange/api.py

```diff
@@ -285,6 +287,41 @@ def get_threat_updates(
         url = f"{self._base_url}/{privacy_group}/threat_updates/"
         return _CursoredResponse(self, url, params, decode_fn=decode_fn)
 
+    def get_threat_privacy_groups_member(
```

Comment:
For HMA we will need to further filter this list to privacy groups that have collaboration functionality enabled (can use `\threat_updates` endpoint). Do you envision this functionality going here? separate function? 

Could be in a later pr

Reply:
How would one go about determining whether a PG can use `/threat_updates`? @jeberl